themes/theme-ios11/pages/Product/components/Media/index.jsx

```diff
@@ -24,9 +25,19 @@ const Media = memo((props) => {
     <Fragment>
       {/* MediaSlider feature is currently in BETA testing.
           It should only be used for approved BETA Client Projects */}
-      {isBeta()
-        ? <MediaSlider productId={variantId || productId} aria-hidden={ariaHidden} />
-        : <ImageSlider productId={productId} variantId={variantId} aria-hidden={ariaHidden} />
+      {
+        isBeta() ? (
+          <ProductMediaSlider
+            productId={variantId || productId}
```

Comment:
These lines differ to `themes/theme-gmd/pages/Product/components/Media/index.jsx`. Please check.

Reply:
Done